core/src/main/java/com/netflix/iceberg/TableProperties.java

```diff
@@ -62,6 +62,9 @@
   public static final String SPLIT_LOOKBACK = "read.split.planning-lookback";
   public static final int SPLIT_LOOKBACK_DEFAULT = 10;
 
+  public static final String SPLIT_EXPENSIVE_TASK_FIRST_ENABLED = "read.split.planning-expensive-task-first.enabled";
```

Comment:
What is the value of making this a table setting? We think that there are cases where it is valuable. Are there also cases where you would want it turned off?

Reply:
I was thinking along the lines to introduce a custom Comparator later which users can configure as a table setting. I thought maintaining all configs pertaining to bin packing could be kept together. I've removed this config for now as I see it adds no value stuff with it as is.